tests/Unit/AuthenticationTest.php

```
@@ -0,0 +1,111 @@
+    public function it_returns_token_substring_for_display(): void
+    {
+        // Arrange
+        $fullToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c';
+        
+        // Act
+        $shortToken = substr($fullToken, 0, 20);
+
+        // Assert
+        $this->assertEquals('eyJhbGciOiJIUzI1NiIs', $shortToken);
+        $this->assertEquals(20, strlen($shortToken));
+    }
```

PHPMD (2.15.0)

The method it_returns_token_substring_for_display is not named in camelCase. (undefined)

(CamelCaseMethodName)